examples/bq-email-exports/README.md

```diff
@@ -1,22 +1,32 @@
 # Automated BigQuery Exports via Email
 
-This solution enables users to regularly send BigQuery export results via email. The end users will get a scheduled email with a link to a Google Cloud Storage [signed URL](https://cloud.google.com/storage/docs/access-control/signed-urls), from which they can view query results as a JSON file.
+This serverless solution enables users to regularly send BigQuery export results via email. The end users will get a scheduled email with a link to a Google Cloud Storage [signed URL](https://cloud.google.com/storage/docs/access-control/signed-urls), from which they can view query results as a JSON, CSV, or Avro file.
 
 The functional steps are listed here:
 
-**Cloud Scheduler:** A [Cloud Scheduler](https://cloud.google.com/scheduler) job invokes the Pub/Sub topic to schedule the email export periodically. The job will take a payload which will include the configurations to run the query and send the email.
-**Pub/Sub:** A [Pub/Sub](https://cloud.google.com/pubsub) topic triggers the Cloud Function.  
-**Cloud Function:** A [Cloud Function](https://cloud.google.com/functions) subscribes to the Pub/Sub topic and runs the code calling the BigQuery and Cloud Storage APIs. 
-**BigQuery:** The [BigQuery API](https://cloud.google.com/bigquery/docs/reference/rest) generates the query results, stores them in a table, and then exports the results as a JSON file into Cloud Storage.  
-**Cloud Storage:** A [Cloud Storage](https://cloud.google.com/storage/) bucket stores the JSON file. The Cloud Storage API generates a signed URL for the JSON file.  
+**Cloud Scheduler:** A [Cloud Scheduler](https://cloud.google.com/scheduler) job defines the time frame between periodic email exports.
+
+**Pub/Sub #1:** A [Pub/Sub](https://cloud.google.com/pubsub) topic is triggered by Cloud Scheduler.
+
+**Cloud Function #1:** A [Cloud Function](https://cloud.google.com/functions) runs the BigQuery query on an [anonymous table](https://cloud.google.com/bigquery/docs/
```

Comment:
Hmm. I think this cloud function can query any kind of table? it's actually the second cloud function that leverages this anonymous table.

Reply:
Right - the first Cloud Function *writes* results to an anonymous table, but can query any type of table. I'll make that more clear